### DISCUSSION TITLE
Scale weather widget for high dpi displays

### DISCUSSION THREAD

**Discussion by chip33** [valinet/ExplorerPatcher#1033] [Ideas]
My screen is 4k @ 350% display scaling and the weather widget takes up almost 2/3 of the display width.
Could there be some kind of size or scale option in Properties?
It would be neat if they could fit with the sizes of the Start Menu.

**Comment by valinet** [comment]
Okay, I have implemented such an option in the latest pre-release.

**Reply by chip33** [reply]
It seems I missed that clicking on the weather widget automatically closes the start menu and vice versa! 
I don't know how I missed that, so there was no need for the size to match the start menu at all. 
Thank you :)

**Reply by chip33** [reply]
There was a small thing I noticed which may not be worth making a post about.
If Start Menu is open, right clicking on taskbar and opening Task Manager auto closes the Start Menu, whereas if you right click taskbar and select EP Properties  the Properties window opens behind the Start Menu.
If you think it is worth auto closing the Start Menu in this case I will open it as an idea .

**Reply by valinet** [reply]
Not that easy to do, I am aiming for a functional implementation, not perfection. It's fine.

**Reply by chip33** [reply]
> Not that easy to do, I am aiming for a functional implementation, not perfection. It's fine.

Yep, cool.  Zoom looks great at 67%.
